pkg/karmadactl/util/idempotency.go

```diff
@@ -0,0 +1,160 @@
+package util
```

Comment:
What's the meaning of the file name `idempotency.go`? 
Should we move these functions to `pkg/util`?

Reply:
This should be from https://github.com/kubernetes/kubernetes/blob/20936e51b80255bbb28a4eed5a814121dac6149e/cmd/kubeadm/app/util/apiclient/idempotency.go

I think it is more applied to `karmadactl` that doesn't call `update` frequently.

Reply:
that's ok. no big deal.